README.md

```diff
@@ -1460,7 +1457,11 @@ itself.
 > int main() {
 >     CLI::App app;
 >     // ...
->     CLI11_PARSE(app);
```

Comment:
The point of `CLI11_PARSE(one arg)` was to make this simpler to type, encouraging people to support unicode. This makes this "short" but magical method four lines long instead! It would better to just use the more reliable and safe method then (`argv = app.ensure_utf8(argv);`), which is only one line longer.